doc/nrfxlib/conf.py

```diff
@@ -27,7 +27,7 @@
 project = "nrfxlib"
 copyright = "2019-2023, Nordic Semiconductor"
```

Comment:
Should copyright date be updated to 2024

Reply:
Updated the copyright year.